sdks/python/apache_beam/runners/portability/local_job_service_main.py

```diff
@@ -155,7 +155,7 @@ def run(argv):
       os.unlink(options.port_file)
 
 
-def serve(msg, job_servicer):
```

Comment:
if i understood correctly "job_servicer" is unused for now, not sure it indicates this paremeter is redundant.

Reply:
i did it because `serve` sounds like its starting the server while its only logging in a loop, it was confusing,  so i removed `job_servicer` to make it a bit clearer that its not starting the server

Reply:
im open to reverting this change, or maybe rename the serve function